contents/courses.tsx

```diff
@@ -12,18 +12,18 @@ export const coursesExplicit: Record<CourseType, CourseDataExplicit> = {
     level: `Tous niveaux`,
     group: `4-5 personnes`,
     duration: `1h15`,
-    price: `330 € les 30 séances ou 120 € les 10 séances ou 15 € / scéance + cotisation association 10 € / personne ou 15 € / famille`,
+    price: `360 € les 30 séances ou 130 € les 10 séances ou 15 € / scéance + cotisation annuelle association 15 € / personne ou 20 € / famille`,
```

Comment:
```suggestion
    price: `360 € les 30 séances ou 130 € les 10 séances ou 15 € / séance + cotisation annuelle association 15 € / personne ou 20 € / famille`,
```
Celui-là je suppose ? Je vais le corriger.

Reply:
yes